src/data.cpp

```diff
@@ -871,7 +936,11 @@ NOVAR80,
 NOVAR80,
 // Feature 0D:
 /*Related:*/ 0x0D,
-NOVAR80,
+DWORD(0x40), DWORD(0x48),
```

Comment:
Isn't feature 0D "airports"? I can't find a 48 VA2 result in either the specs or the code. And I wonder where the rest comes from. I do agree that 40 seems to be missing here.

Reply:
Airports use [https://newgrf-specs.tt-wiki.net/wiki/VariationalAction2/BaseStation](base station vars)